### PR TITLE
Update 3DS Flow to FPTI Analytics

### PR DESCRIPTION
**Pull request by sarahkoop**
### Summary of changes

 - Update 3DS flows with FPTI analytics
 - Includes secondary analytics events that may become extra info on conversion events in the future
 - Removed challenge succeeded/failed analytics that don't make sense to track in Android flow, since they are done outside of the `ThreeDSecureClient`

### Checklist

 - ~[ ] Added a changelog entry~
 - [x] Relevant test coverage

### Authors
@sarahkoop

### CODE REVIEW

File: ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureClient.java [hunk 1]
```diff
@@ -285,48 +264,54 @@ public void tokenize(ThreeDSecurePaymentAuthResult paymentAuthResult,
...
-                                        sendLiabilityShiftedAnalytics(threeDSecureResult1);
-                                        callback.onThreeDSecureResult(new ThreeDSecureResult.Success(threeDSecureResult1.getThreeDSecureNonce()));
+                                        braintreeClient.sendAnalyticsEvent(ThreeDSecureAnalytics.JWT_AUTH_SUCCEEDED);
+                                        callbackTokenizeSuccess(callback, new ThreeDSecureResult.Success(threeDSecureResult1.getThreeDSecureNonce()));
                                     }
                                 } else if (error != null) {
-                                    braintreeClient.sendAnalyticsEvent(
-                                            "three-d-secure.verification-flow.upgrade-payment-method.errored");
-                                    callback.onThreeDSecureResult(new ThreeDSecureResult.Failure(error, null));
+                                    braintreeClient.sendAnalyticsEvent(ThreeDSecureAnalytics.JWT_AUTH_FAILED);
```
Comment (tdchow) [inline ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureClient.java:274]: Similar to my comment on the other analytics PR - would we want to send the failure type as a param instead?

**Review by tdchow — APPROVED** [review]

**Review by agedd — APPROVED** [review]
lgtm! 🚀

File: ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureClient.java (hunk 1 above)
Comment (tdchow) [reply ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureClient.java:274]: resolving - similar to this thread: https://github.com/braintree/braintree_android/pull/920#discussion_r1488551108

Comment (sarahkoop) [reply ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureClient.java:274]: Resolved in other PR - keeping these events to make it easier to see what extra info should be added to each of these events - they can be removed when the extra params are added to analytics

**Pull request merged**